src/dotnet-scaffolding/dotnet-scaffold-aspnet/Commands/API/MinimalApi/MinimalApiCommand.cs

```diff
@@ -339,6 +340,12 @@ private void InstallEfPackages(MinimalApiSettings commandSettings)
             packageList.Add(projectPackageName);
         }
 
-        CommandHelpers.InstallPackages(_logger, commandSettings.Project, commandSettings.Prerelease, packageList);
+        await new AddPackagesStep
```

Comment:
The code won't live like this for very long because of all the refactoring we're doing, but I appreciate the effort at conciseness combining all this into one 🤣

Reply:
yes nwnw